edx_credentials_themes/static/mitpe/certificates/css/mitpe.certificate.style-ltr.css

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
It looks like the new config is placing the files at `.../mitpe/certificates/css/...` and the `base.html` is looking for them at `.../mitpe/css/...`. How much easier would this be if we just delete the MITPE stuff before attempting this? It's a pretty risk-less PR to make and we should just ping George for a :+1: . Then you won't be trying to deal with the "old way" and the "new way" at the same time.

Reply:
I will look into this now.